lib/OpenQA/Setup.pm

```diff
@@ -224,6 +228,16 @@ sub update_config {
     }
 }
 
+sub prepare_settings_ui_keys {
+    my ($app) = shift;
+    my @link_keys = split ',', $app->config->{job_settings_ui}->{keys_to_render_as_links};
+    $app->config->{settings_ui_links} = {map { $_ => 1 } @link_keys};
+    # $app->config->{settings_ui_links} //= {};
+    # my @link_keys = split ",", $app->config->{job_settings_ui}->{keys_to_render_as_links};
+    # my %link_keys_hash = map { $_ => 1 } @link_keys;
+    # $app->config->{settings_ui_links} = \%link_keys_hash if (@link_keys);
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
yea sorry. i submitted forgetting to remove it.